cves/CVE-2008-2364.yml

```diff
@@ -54,7 +58,7 @@ fixes:
    - commit: 52d29a43ce04c396da3ae26c16c04b47a9f77ff7
      note:
```

Comment:
Maybe leave some notes for these other fix commits.

Reply:
The changes in those commits were simply status updates referring to the last commit. However I added in some basic information about the commits anyway.